src/lib/util.js

```diff
@@ -294,6 +297,24 @@ export function prefetchAssets(urls, preload = false) {
     });
 }
 
+/**
+ * Method to fetch a given representation as a blob via an authenticated content URL.
+ * Useful for determining the true state of a "successful" representation
+ *
+ * @param {string} contentUrl - a representation's authenticated content URL
+ * @return {Promise} - Resolves with the representation data, rejects with a deleted rep error
+ */
+export function fetchRepresentationAsBlob(contentUrl) {
```

Comment:
I added this to util because the DocPreloader will need it when fetching first page images. That class lives outside of the normal viewer inheritance hierarchy.

Reply:
Is it possible for us to share the response handler function only, rather than the api call itself? That seems to fit more as a util, to me.

Reply:
It's possible, but I like the idea of keeping the response type in one place so that it's not repeated every time we use it.